openapi.yaml

```diff
@@ -21,6 +21,8 @@ paths:
           schema:
             type: integer
             format: int32
+            minimum: 1
+            maximum: 100
```

Comment:
Stimmt das so? Haben wir im Backend ein max von 100?

Reply:
Umgesetzt ist das soweit ich weiß noch nicht. Von mir aus können wir das auch raus schmeißen. In der description steht aber auch schon `(max 100)`. Da dachte ich, dass wir es auch so setzten können.

Reply:
joa, lass das sonst aus der spec rausschmeißen

Reply:
und halt dann auch aus der description

Reply:
Ok, mach ich unten dann auch.